trigger/payloads.go

```diff
@@ -470,8 +470,11 @@ func buildLink(uiBase string, buildID string, hasPatch bool) string {
 	return url
 }
 
-func versionLink(uiBase string, versionID string, hasPatch bool) string {
+func versionLink(uiBase string, versionID string, hasPatch bool, isChild bool) string {
```

Comment:
It's not great to have multiple booleans in a row for function signatures, since it's easy to get these mixed up, so it's probably worth an input struct now.

Reply:
true, fixed.